src/solver/conv_wino_fury_RxS.cpp

```diff
@@ -0,0 +1,457 @@
+
+          o_H_clip_bot_neg{static_cast<int16_t>(o_H_clip)},
+          o_W_clip_bot_neg{static_cast<int16_t>(o_W_clip)},
+
+          d_H_window{static_cast<uint16_t>(H)},
+          d_W_window{static_cast<uint16_t>(W)},
+
+          o_H_window{static_cast<uint16_t>(out_h)},
+          o_W_window{static_cast<uint16_t>(out_w)},
+
+          n_groups{static_cast<uint32_t>(groups)}
+    {
+        is_applicable = problem.IsFp16() && problem.Is2d() && problem.GetGroupCount() == 1 &&
+                        problem.GetInLayout() == "NCHW" && !problem.direction.IsBackwardWrW();
+    }
+
+    float GetWTI() const { return -2.f; } // unknown
+
+    bool isApplicable() const
+    {
+        return is_applicable
+               // clang-format off
+               && U == 1
+               && V == 1
+               && input_stride_h == 1
+               && input_stride_w == 1
+               && filter_stride_h == 1
+               && filter_stride_w == 1
+               && C <= 16
+               && K <= 16
+               && S <= 3
+               && R <= 3
+               && N < std::numeric_limits<uint16_t>::max()
+               && pad_h <= std::numeric_limits<int16_t>::max()
+               && pad_h >= std::numeric_limits<int16_t>::lowest()
+               && pad_w <= std::numeric_limits<int16_t>::max()
+               && pad_w >= std::numeric_limits<int16_t>::lowest()
+               && n_groups < std::numeric_limits<uint16_t>::max()
+               && static_cast<uint32_t>(out_Hs) == Hs
+               && static_cast<uint32_t>(out_We) == We
+               && static_cast<int32_t>(d_H_clip_bot_neg) == d_H_clip
+               && static_cast<int32_t>(d_W_clip_bot_neg) == d_W_clip
+               && static_cast<int32_t>(o_H_clip_bot_neg) == o_H_clip
+               && static_cast<int32_t>(o_W_clip_bot_neg) == o_W_clip
+               && static_cast<uint32_t>(d_H_window) == H
+               && static_cast<uint32_t>(d_W_window) == W
+               && static_cast<uint32_t>(o_H_window) == out_h
+               && static_cast<uint32_t>(o_W_window) == out_w;
+        // clang-format on
+    }
+};
+
+} // namespace
+
+template <uint32_t Winodata, uint32_t Winofilter>
+bool ConvWinoFuryRxS<Winodata, Winofilter>::IsApplicable(const ConvolutionContext& ctx,
+                                                         const ProblemDescription& problem) const
+{
+    if(is2x3() && miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_FURY_RXS_F2X3{}))
+        return false;
+
+    if(is3x2() && miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_FURY_RXS_F3X2{}))
+        return false;
+
+    if(!ctx.use_asm_kernels)
+        return false;
+    if(!ctx.rmv.IsV3())
+        return false;
+
+    const auto target = ctx.GetStream().GetTargetProperties();
+    if(target.Xnack() && *target.Xnack())
+        return false;
+
+    const auto name = ctx.GetStream().GetDeviceName();
+    if(!(StartsWith(name, "gfx1100") || StartsWith(name, "gfx1101")))
+        return false;
```

Comment:
We do not need StartsWith when comparing for equality.

```suggestion
    if(!(name == "gfx1100" || name == "gfx1101"))
        return false;
```